great_expectations/expectations/expectation.py

```diff
@@ -1752,6 +1752,47 @@ def _isclose(
         return np.isclose(operand_a_as_number, operand_b_as_number)
 
 
+class QueryExpectation(TableExpectation, ABC):
+
```

Comment:
Let us add a docstring here in the same fashion as we do for other `Expectation` types.

Reply:
Happy to add a docstring here! Worth noting that currently *only* the `MetaExpectation` and base `Expectation` classes have docstrings.